daffodil-test/src/test/resources/org/apache/daffodil/section02/schema_definition_errors/SchemaDefinitionErrors.tdml

```diff
@@ -203,9 +203,11 @@
                        description="">
     <tdml:document><![CDATA[test]]></tdml:document>
 
-    <tdml:errors>
-      <tdml:error />
-    </tdml:errors>
+    <tdml:infoset>
+      <tdml:dfdlInfoset>
+        <elem>test</elem>
+      </tdml:dfdlInfoset>
+    </tdml:infoset>
```

Comment:
Just want to point out that if there is not a ``<tdml:warning>`` element in a test, the TDML runner doesn't care if there are warnings or not. The TMDL Runner only requires that warnings exist if the ``<tdml:warnigns>`` element exists. So unfortunately there's no way test that no warnings were output. Probably still worth having this, since some point we will want to add that feature and if we are ignoring a warning here, that would be caught some day.

Reply:
Ah, ok.  Handling the diagnostics output on stderr in the cli tests must have had me thinking this was checked, but I see what you mean now.